examples/python/arkitscenes/main.py

```diff
@@ -0,0 +1,198 @@
+    depth_dir = recording_path / "lowres_depth"
+    intrinsics_dir = recording_path / "lowres_wide_intrinsics"
+    traj_path = recording_path / "lowres_wide.traj"
+
+    frame_ids = [x.name for x in sorted(depth_dir.iterdir())]
+    frame_ids = [x.split(".png")[0].split("_")[1] for x in frame_ids]
+    frame_ids.sort()
+
+    poses_from_traj = {}
+    with open(traj_path, "r", encoding="utf-8") as f:
+        traj = f.readlines()
+
+    for line in traj:
+        ts, Rt = traj_string_to_matrix(line)
+        # round timestamp to 3 decimal places
+        ts = f"{round(float(ts), 3):.3f}"
+        poses_from_traj[ts] = Rt
+
+    ply_path = recording_path / f"{recording_path.stem}_3dod_mesh.ply"
+    bbox_annotations_path = recording_path / f"{recording_path.stem}_3dod_annotation.json"
+    annotation = load_json(bbox_annotations_path)
+    log_annotated_bboxes(annotation)
+
+    # TODO(pablovela5620): Wait for resolution of either #1570 or #1571 for textured mesh
+    # for now just use the untextered/uncolored mesh
+    # # convert ply to obj
+    # obj_path = recording_path / f"{recording_path.stem}_3dod_mesh.obj"
+    # if not obj_path.exists():
+    #     mesh = o3d.io.read_triangle_mesh(str(ply_path))
+    #     o3d.io.write_triangle_mesh(str(obj_path), mesh)
+
+    ply = trimesh.load(str(ply_path))
+    verts = ply.vertices
+    faces = ply.faces
+
+    # Log the mesh
+    rr.log_mesh("world/mesh_log", positions=verts, indices=faces, timeless=True)
+
+    # with open(obj_path, mode="rb") as file:
+    #     rr.log_mesh_file("world/mesh_log_file", MeshFormat.OBJ, file.read(), timeless=True)
+
+    # To avoid logging image frames in the beginning that dont' have a trajectory
+    # This causes the camera to expand in the beginning otherwise
+    init_traj_found = False
+    for frame_id in tqdm(frame_ids):
+        rr.set_time_seconds("time", float(frame_id))
+        bgr = cv2.imread(f"{image_dir}/{video_id}_{frame_id}.png")
+        depth = cv2.imread(f"{depth_dir}/{video_id}_{frame_id}.png", cv2.IMREAD_ANYDEPTH)
+        # Log the camera transforms:
+        if str(frame_id) in poses_from_traj:
+            if not init_traj_found:
+                init_traj_found = True
+            intrinsic_fn = intrinsics_dir / f"{video_id}_{frame_id}.pincam"
+            w, h, fx, fy, cx, cy = np.loadtxt(intrinsic_fn)
+            intrinsic = np.array([[fx, 0, cx], [0, fy, cy], [0, 0, 1]])
+            frame_pose = np.array(poses_from_traj[str(frame_id)])
+
+            rot_matrix = frame_pose[:3, :3]
+            translation = frame_pose[:3, 3]
+
+            rot = R.from_matrix(rot_matrix[0:3, 0:3])
+            quaternion = rot.as_quat()
+
+            # TODO(pablovela5620): Fix orientation for portrait captures in 2D view once #1387 is closed.
+            rr.log_rigid3(
+                "world/camera",
+                parent_from_child=(translation, quaternion),
+                xyz="RDF",  # X=Right, Y=Down, Z=Forward
+            )
```

Comment:
See earlier comment regarding converting to quaternion directly and using `child_from_parent` `parent_from_child`